.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs

```diff
@@ -248,7 +407,25 @@ private DateTimeResolutionResult ParseEach(IDateTimeExtractor extractor, IDateTi
             {
                 var pr = parser.Parse(ers[0], refDate);
 
-                ret = SetHandler.ResolveSet(ref ret, pr.TimexStr);
+                if ((config.Options & DateTimeOptions.TasksMode) != 0)
+                {
+                    if (match.Success)
+                    {
+                        pr.TimexStr = TasksModeSetHandler.TasksModeTimexIntervalExt(pr.TimexStr);
+                    }
+
+                    if (match.Groups["other"].Success)
+                    {
+                        pr.TimexStr = pr.TimexStr.Replace("P1", "P2");
```

Comment:
What does this replace mean? This should be a method with a clear name, and probably P1 and P2 should be constants.

Reply:
Replaced with proper method and code comment.